indexing-hadoop/src/test/java/io/druid/indexer/InputRowSerdeTest.java

```diff
@@ -88,5 +90,29 @@ public void testSerde()
     Assert.assertEquals(5.0f, out.getFloatMetric("m1out"), 0.00001);
     Assert.assertEquals(100L, out.getLongMetric("m2out"));
     Assert.assertEquals(1, ((HyperLogLogCollector)out.getRaw("m3out")).estimateCardinality(), 0.001);
+    Assert.assertEquals(0L, out.getLongMetric("unparseable"));
+
+  }
+
+  @Test
+  public void testThrowParseExceptions(){
+    InputRow in = new MapBasedInputRow(
+        timestamp,
+        dims,
+        event
+    );
+    AggregatorFactory[] aggregatorFactories = new AggregatorFactory[] {
+        new DoubleSumAggregatorFactory("agg_non_existing", "agg_non_existing_in"),
+        new DoubleSumAggregatorFactory("m1out", "m1"),
+        new LongSumAggregatorFactory("m2out", "m2"),
+        new HyperUniquesAggregatorFactory("m3out", "m3"),
+        new LongSumAggregatorFactory("unparseable", "m3") // Unparseable from String to Long
+    };
+    try {
+      InputRowSerde.toBytes(in, aggregatorFactories, true);
+      Assert.fail("Should have thrown parseException");
+    } catch (ParseException e){
```

Comment:
Better to use the `org.junit.rules.ExpectedException` `@Rule` or use `@Test(expected = ParseException.class)`